objects_test.go

```diff
@@ -603,6 +603,39 @@ var _ = Describe("Objects", func() {
 			})
 		})
 
+		Context("RecordMX object", func() {
+			fqdn := "test.example.com"
+			mx := "example.com"
+			dnsView := "default"
+			priority := 10
+			ttl := uint32(70)
+			useTtl := true
+			comment := "test comment"
+			eas := EA{"Country": "test"}
+
+			rm := NewRecordMX(RecordMX{
+				Fqdn:     fqdn,
+				MX:       mx,
+				View:     dnsView,
+				Priority: priority,
+				Ttl:      ttl,
+				UseTtl:   useTtl,
+				Comment:  comment,
+				Ea:       eas,
+			})
+
+			It("should set fields correctly", func() {
+				Expect(rm.Fqdn).To(Equal(fqdn))
+				Expect(rm.MX).To(Equal(mx))
+				Expect(rm.View).To(Equal(dnsView))
```

Comment:
What about other fields of 'rm' structure? I mean: Priority, Ttl, UseTtl, Comment, Ea.